examples/caps_wing/10_blade_stiffened.py

```diff
@@ -0,0 +1,82 @@
+"""
+Sean Engelstad, March 2024
+GT SMDO Lab, Dr. Graeme Kennedy
+
+Based off of Alasdair Christan Gray's Mach wing blade stiffened example in TACS : examples/mach_tutorial_wing
```

Comment:
Pedantic I know, but my surname is Christison Gray :stuck_out_tongue:

Reply:
I fixed it